src/test/java/com/blackducksoftware/integration/hub/alert/TestProperties.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (C) 2017 Black Duck Software Inc.
+ * http://www.blackducksoftware.com/
+ * All rights reserved.
+ *
+ * This software is the confidential and proprietary information of
+ * Black Duck Software ("Confidential Information"). You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Black Duck Software.
+ */
+package com.blackducksoftware.integration.hub.alert;
+
+import java.util.Properties;
+
+import org.junit.Assume;
+
+public class TestProperties {
+    private final ResourceLoader resourceLoader;
+    private Properties properties;
+
+    public TestProperties() {
+        resourceLoader = new ResourceLoader();
+    }
+
+    public Properties getProperties() {
+        if (properties == null) {
+            try {
+                // TODO add additional locations to check for properties
```

Comment:
We should parameterize this method so that the location can be passed in, create a default version that uses that constant.